Client/Frontend/Browser/FaviconManager.swift

```diff
@@ -17,7 +17,7 @@ class FaviconManager : BrowserHelper {
         if let path = NSBundle.mainBundle().pathForResource("Favicons", ofType: "js") {
             if let source = NSString(contentsOfFile: path, encoding: NSUTF8StringEncoding, error: nil) as? String {
                 var userScript = WKUserScript(source: source, injectionTime: WKUserScriptInjectionTime.AtDocumentEnd, forMainFrameOnly: true)
-                browser.webView.configuration.userContentController.addUserScript(userScript)
+                browser.webView?.configuration.userContentController.addUserScript(userScript)
```

Comment:
Good idea. At one point I had the tabs doing some jumps through hoops to store and restore these, but I just decided it was easier to use the callbacks and let these do it themselves. I'll make this non-optional.